boxoffice/models/invoice.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from decimal import Decimal
+from boxoffice.models import db, BaseMixin, UuidMixin
+from sqlalchemy import event
+from sqlalchemy import sql, func
+from sqlalchemy.ext.orderinglist import ordering_list
+
+__all__ = ['Invoice', 'InvoiceLineItem']
+
+
+def get_latest_invoice_no(organization):
+    """
+    Returns the last invoice number used, 0 if no order has ben invoiced yet
+    """
+    query = db.session.query(sql.functions.max(Invoice.invoice_no))\
+        .filter(Invoice.organization == organization)
+    if organization.fy_start_at:
```

Comment:
`fy_start_at` is now computed dynamicall based on `fy_start_month`, `fy_start_day` and `fy_timezone`.